### PR TITLE
chore(core): update copy for new version available toast

### PR DESCRIPTION
**Pull request by runeb**

### CODE REVIEW

**Comment by cngonzalez** [comment]
to create this new UI 
Screenshot 2024-07-10 at 2 04 55 PM

File: packages/sanity/src/core/i18n/bundles/studio.ts [hunk 1]
```diff
@@ -1085,12 +1085,10 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   'new-document.open-dialog-aria-label': 'Create new document',
   /** Title for "Create new document" dialog */
   'new-document.title': 'Create new document',
-  /** More detailed alert text letting user know they have an out-of-date version and should reload */
-  'package-version.new-package-available.description': 'Simply reload to use the new version.',
   /** Label for button that will make the browser reload when users' studio version is out-of-date */
-  'package-version.new-package-available.reload-button': 'Reload',
+  'package-version.new-package-available.reload-button': 'Push to reload',
```
Comment (binoy14) [inline packages/sanity/src/core/i18n/bundles/studio.ts:1089]: Should this be `Click` ? `Push to reload` sounds weird to me

**Review by binoy14 — COMMENTED** [review]
Comment regarding copy, looks good otherwise

File: packages/sanity/src/core/i18n/bundles/studio.ts (hunk 1 above)
Comment (rexxars) [reply packages/sanity/src/core/i18n/bundles/studio.ts:1089]: Knut suggested "Push" to be less user input interface biased (eg "click" is mouse/trackpad, "tap" is touch). No strong feelings on my side.

**Review by binoy14 — APPROVED** [review]
Approved copy on Knut's suggestion

**Pull request merged**